packages/calypso-doctor/evaluations/node-memory.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
 
 		const currentValue = Number( match[ 1 ] );
 		const desiredValue = getMemInMb() * 0.75;
-		if ( desiredValue !== currentValue ) {
-			fail( `Memory set to ${ currentValue } mb, ${ desiredValue } mb expected` );
+		if ( desiredValue > currentValue ) {
```

Comment:
This should be the other way around: if current allocated memory is `<` than the "ideal", then fail the test. (75% of available ram, I just pull that number out of thin air tbh)

Reply:
Alright, updated. I think it was still working the same way, but this is more readable

Reply:
Oh I see it now. Thanks for changing it, I think it is more readable too.

Reply:
To clear any potential leftover confusion (noticing those confused reactions :P), the thinking is that if the doctor wants to have more memory than is currently designated (e.g. "desired > current"), then the check fails. (Which is the same as "current < desired", e.g. there is less current memory than there should be)